paddle/fluid/eager/backward.cc

```diff
@@ -241,12 +493,39 @@ void RunBackward(const std::vector<paddle::experimental::Tensor>& tensors,
                        paddle::platform::errors::Fatal(
                            "Detected in-degree value smaller than zero."
                            "Node's in-degree cannot be negative"));
-        if (node_in_degree_map[next_node] == 0) {
-          queue.emplace(std::move(next_node));
+
+        bool is_potential_stop_node = false;
```

Comment:
bool is_potential_stop_node = potential_stop_nodes.count(next_node);

Reply:
Thx, it's better now.